tests/upgrade/legacy_to_postgres_run.sh

```diff
@@ -1,12 +1,17 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC1091
 
-# TODO(ROX-22872): Remove this test
 set -euo pipefail
 
 # Tests upgrade to Postgres.
+# NOTE:  This test will upgrade RocksDB to Postgres at release 4.4.
```

Comment:
Consider explaining the `3.74` release, like the major version switch etc., for posterity.